### PR TITLE
Localize content/en/docs/reference/glossary/helm-chart.md in hindi

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>
fixes #37078

### CODE REVIEW

**Review by bishal7679 — APPROVED** [review]
Taking ref from #37078
/lgtm

File: content/hi/docs/reference/glossary/helm-chart.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+  - tool
+---
+
+हेल्म चार्ट(Helm Chart) पूर्व-कॉन्फ़िगरड़ (pre-configured) कुबेरनेट्स संसाधनों का एक पैकेज है जिसे हेल्म टूल के माध्यम से प्रबंधित किया जा सकता है।
+
+<!--more-->
+
+कुबेरनेट्स एप्लिकेशन बनाने और साझा करने के लिए चार्ट एक प्रतिलिपि प्रस्तुत करने योग्य तरीका प्रदान करते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/helm-chart.md:17]:
```suggestion
चार्ट्स कुबेरनेट्स एप्लिकेशन बनाने और साझा करने के लिए एक पुनरुत्पादनीय तरीका प्रदान करते हैं।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/helm-chart.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+date: 2018-04-12
+full_link: https://helm.sh/docs/topics/charts/
+short_description: >
+  हेल्म चार्ट(Helm Chart) पूर्व-कॉन्फ़िगर(pre-configured) किए गए कुबेरनेट्स संसाधनों का एक पैकेज है जिसे हेल्म टूल के माध्यम से प्रबंधित किया जा सकता है।
+aka:
+tags:
+  - tool
+---
+
+हेल्म चार्ट(Helm Chart) पूर्व-कॉन्फ़िगरड़ (pre-configured) कुबेरनेट्स संसाधनों का एक पैकेज है जिसे हेल्म टूल के माध्यम से प्रबंधित किया जा सकता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/helm-chart.md:13]:
```suggestion
हेल्म चार्ट(Helm Chart) पूर्व-कॉन्फ़िगर (pre-configured) किए गए कुबेरनेट्स संसाधनों का एक पैकेज है जिसे हेल्म टूल के माध्यम से प्रबंधित किया जा सकता है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Review by bishal7679 — APPROVED** [review]
/lgtm

**Pull request merged**